content/03.results.md

```diff
@@ -168,11 +167,11 @@ We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medu
 By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
 Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**).
 The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
-We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas (high-grade glioma astrocytomas) and in 28.2% (N = 11/39) of diffuse midline gliomas (**Figure {@fig:Fig3}D**).
-We also found evidence of chromothripsis in over 15% of sarcomas, metastatic secondary tumors, embryonal tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, and ependymomas, highlighting the genomic instability and complexity of these pediatric brain tumors.
+We identified chromothripsis events in 28% (N = 11/39) of diffuse midline gliomas and 40% (N = 19/48) of non-midline HGGs (other HGGs) (**Figure {@fig:Fig3}D**).
+We also found evidence of chromothripsis in over 15% of sarcomas, PXAs, metastatic secondary tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, ependymomas, medulloblastomas, ATRTs, and other embryonal tumors, highlighting the genomic instability and complexity of these pediatric brain tumors.
 
 We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures from the RefSig database [@doi:10.1038/s43018-020-0027-5] across samples (**Figure {@fig:Fig3}E** and **Figure {@fig:S4}A**).
-Stage 0 and/or 1 tumors characterized by low TMBs (**Figure {@fig:S2}G**) such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas were dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
+Stage 0 and/or 1 tumors characterized by low TMBs (**Figure {@fig:S2}H**) such as pilocytic astrocytomas, gangliogliomas, other LGGs, and craniopharyngiomas, were dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
```

Comment:
This figure S2H doesn't contain information about Signature 1. Maybe this should be referencing Figure S4 as well or instead.

Reply:
Oh yes this was referring to Tmb but I should add S4 at the end referring to sigs. Thanks!